tests/test_expectation_maximization.py

```diff
@@ -23,24 +23,40 @@ def n_particles():
 
 
 @pytest.fixture
-def test_ir(n_pix, n_particles):
+def rand_angle_list(n_particles):
+    """Get random astigmatism angle between 0 and 2pi."""
+    return np.random.uniform(low=0, high=2 * np.pi, size=(n_particles,))
+
+
+@pytest.fixture
+def rand_defocus(n_particles):
+    """Get random defocus values between 0.5 and 2.5."""
+    return np.random.uniform(low=0.5, high=2.5, size=(n_particles,))
+
+
+@pytest.fixture
+def test_ir(n_pix, n_particles, rand_defocus, rand_angle_list):
     """Instantiate IterativeRefinement class for testing."""
-    ex_ctf = {
-        "s": np.ones((n_pix, n_pix)),
-        "a": np.ones((n_pix, n_pix)),
-        "def1": 1.0,
-        "def2": 1.0,
-        "angast": 0.1,
-        "kv": 0.1,
-        "cs": 0.1,
-        "bf": 0.1,
-        "lp": 0.1,
+    pixels = n_pix
+    ctf_info = {
+        "amplitude_contrast": 0.1,
+        "b_factor": 0.0,
+        "batch_size": n_particles,
+        "cs": 2.7,
+        "ctf_size": pixels,
+        "kv": 0.1968,
```

Comment:
kv could/should be 300. in that case wavelength would be 0.1968.

it's my understanding that `ctf.get_ctf` needs `wavelength` and doesn't use `kv`